FeatureDetect/src/BuildOpenCV.cmake

```diff
@@ -0,0 +1,134 @@
+set(PROJECT_BUILD_DIR ${CMAKE_SOURCE_DIR}/build)
+
+set(OPENCV_VERSION 4.6.0)
+set(OPENCV_MODULES core,calib3d,imgproc,imgcodecs,features2d,highgui)
```

Comment:
Add something like `# TODO: remove unused modules` here not to forget about it

Reply:
done